_datascience/deidentification.md

```diff
@@ -1,114 +1,173 @@
+participate in research, including protections related to data and specimens
+collected from individuals for both primary and secondary research. Research
+with data and specimens that are collected for another purpose (secondary
+research) must be de-identified to be considered not human subjects
+research.[^1]
+
+According to the [U.S. Department of Health and Human
+Services](https://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html#rationale), 
+
+> The process of de-identification, by which identifiers are removed from the
+> health information, mitigates privacy risks to individuals and thereby
+> supports the secondary use of data for comparative effectiveness studies,
+> policy assessment, life sciences research, and other endeavors.
+
+More information about HIPAA compliance is available on [FHCC's IRB
+website](https://extranet.fredhutch.org/en/u/irb/hipaa-compliance.html).
+
+## Methods for creating de-identifed datasets 
+
+The HIPAA Privacy Rule provides two methods for de-identification of PHI. The
+first method, expert determination, refers to expert application and evaluation
+of methods to de-identify data. The second, safe harbor, refers to removal of
+specific identifiers from a data set.
+
+> ⚠️ While it is possible to de-identify your data yourself, unless you are an
+> expert, we suggest that you seek out an expert to help you instead. There are
+> nuances regarding privacy that you may miss without ample experience.
+
+### The "expert determination" method
+
+The HIPAA Privacy Rule defines de-identification by expert determination as a
+process where a person with: 
+
+> appropriate knowledge of and experience with generally accepted statistical
+> and scientific principles and methods for rendering information not
+> individually identifiable,
+
+is able to apply these principles and methods to determine:
+
+>  that the risk is very small that the information could be used, alone or in
+>  combination with other reasonably available information, by an anticipated
+>  recipient to identify an individual who is a subject of the information,
+
+and where this person is able to document:
+
+>  the methods and results of the analysis that justify such determination.
+
+In summary, an expert in methods for de-identification can apply scientific and
+statistical principles to de-identify data as long as the risk of
+re-identification is very small, and must be able to document their methods and
+reasoning. More guidance on the [expert determination method for
+de-identification](https://www.hhs.gov/hipaa/for-professionals/privacy/special-topics/de-identification/index.html#guidancedetermination)
+is on the HHS website.
+
+### The "safe harbor" method
+
+A data set is considered de-identified using the safe harbor method if the following identifiers are removed:
+
+  1. Names
+  2. Geographic subdivisions smaller than the state
```

Comment:
might want to add this: Zip codes may remain in the data if they are shortened to the initial 3 digits and meet certain population threshold criteria.

Reply:
@carriewright11 do you have a reference for that or a link to something?

Reply:
https://www.johndcook.com/blog/2019/08/29/zip3-privacy/

Reply:
thanks! I'm adding all your links to the next wiki page :)